src/components/bulk-email-tool/bulk-email-task-manager/BulkEmailTaskManager.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+import { FormattedMessage } from '@edx/frontend-platform/i18n';
+import BulkEmailContentHistory from './BulkEmailContentHistory';
+import BulkEmailPendingTasks from './BulkEmailPendingTasks';
+import BulkEmailTaskHistory from './BulkEmailTaskHistory';
+
+export default function BulkEmailTaskManager() {
+  return (
+    <div className="px-3">
+      <div>
+        <p className="h3">
```

Comment:
Why not just use an `h3` tag here?

Reply:
I was trying to think about a11y concerns. I wasn't really sure how the different sections were going to get implemented so I just styled it _like_ an H3 for now. I _believe_ that headers need to go from largest to smallest, and that we can only have one h1, but I wasn't sure how the styles were going to fall since we are doing each section in chunks. 

Let me know if I am overthinking this :D.

Reply:
correct! they do go largest -> smallest, but you can skip numbers. There will eventually be an `h1` tag on this page, so setting these to `h3` I think is fine. Otherwise I believe screen readers won't scan them as headers, which I think we'd want in this case.

Reply:
Ah, ok! I catch what you're saying now. Thanks. I'll set this to a header tag. I took a look at the original tool and I think it may be more appropriate to set this as a <h2> then but I think I will keep the h3 styling.